<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rename Project Object</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Object Type:</string>
        </property>
        <property name="buddy">
         <cstring>txtCurrentObject</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtCurrentObject">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Current Name:</string>
        </property>
        <property name="buddy">
         <cstring>txtCurrentName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtCurrentName">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>New Name:</string>
        </property>
        <property name="buddy">
         <cstring>editNewName</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editNewName">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpWhereUsed">
     <property name="title">
      <string>Where Used</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="gridUsage"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblInstructions">
     <property name="text">
      <string>Press OK to change the object name and change everywhere it is used.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblInstructions2">
     <property name="text">
      <string>Press Cancel to exit this dialog without renaming.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
